src/picosvg/svg_meta.py

```diff
@@ -36,10 +36,15 @@ def xlinkns():
     return "http://www.w3.org/1999/xlink"
 
 
+def splitns(name):
+    if "}" in name:
```

Comment:
I guess you know lxml has the QName class for qualified XML names, with which you can split the `namespace` and `localname` attributes:
https://lxml.de/api/lxml.etree.QName-class.html

should we use that here?

Reply:
I always forget that exists!